apps/smartling/lambda/package.json

```diff
@@ -7,7 +7,7 @@
     "build": "rimraf built && tsc",
     "lint": "tslint --project ./tsconfig.json",
     "test": "jest --watch",
-    "test:ci": "jest",
+    "test:ci": "cd ../frontend && npm run build && cd ../lambda && jest",
```

Comment:
nice (not too ugly and readable tbh)

Reply:
Haha not sure about "not too ugly" but the value it adds in decoupling the whole big build for the tests is probably worth it for me.

I mean I might make a small suggestion to make it a bit less hacky by moving this out to a separate task (also to add a bit of clarity) i.e.:

```
"build-frontend": "cd apps/smartling/frontend && npm run build"
"test:ci": "npm run build-frontend && jest"
```

(btw I didn't test to see if you need to cd back in)